bciers/apps/administration/app/components/facilities/FacilitiesForm.tsx

```diff
@@ -58,7 +58,7 @@ export default function FacilitiesForm({
         }
         if (isCreating) {
           router.replace(
-            `/operations/${params.operationId}/facilities/${response.id}?facilitiesTitle=${response.name}`,
+            `/operations/${params.operationId}/facilities/${response.id}?facilities_title=${response.name}`,
             // @ts-ignore
             { shallow: true },
```

Comment:
Not directly related to this PR though I hope we get rid of this `@ts-ignore` at some point since shallow routing does not work with NextJS app router. Sepehr updated the `ContactForm` to use `window.history.replaceState` that the co-author of Next says to use for shallow routing:

https://github.com/vercel/next.js/discussions/48110#discussioncomment-7563979

Reply:
![image](https://github.com/user-attachments/assets/9a4c23ae-4f4e-4a81-96d4-bed290c07f71)